resources/js/components/FormConditions.vue

```diff
@@ -3,22 +3,59 @@ import '/public/vendor/statamic/frontend/js/helpers.js'
 Vue.prototype.Statamic = window.Statamic
 
 export default {
-    props: ['initialData'],
+    props: {
+        initialData: {
+            type: Object,
+            required: true,
+        },
+        translations: {
+            type: Object,
+            default: () => ({}),
+        },
+        callback: {
```

Comment:
Added a callback because in a project you might want to add some functionality like "closing a modal" or conversion tracking.